src/edu/stanford/dlss/was/WasapiCrawlSelector.java

```diff
@@ -7,10 +7,15 @@
 
 public class WasapiCrawlSelector {
 
-  public Map<Integer, List<WasapiFile>> crawlIdToFiles;
+  protected Map<Integer, List<WasapiFile>> crawlIdToFiles = new HashMap<Integer, List<WasapiFile>>();
```

Comment:
I'm all for appropriate visibility ... but curious:  why "protected"?  would "package" work?